gui.py

```
@@ -0,0 +1,550 @@
+        self._run(args)
+
+    def _run_cat(self):
+        ver  = self._val(self.cat_ver)
+        path = self._val(self.cat_path)
+        if not ver or not path:
+            self._warn("Firmware file and inner path are both required."); return
+        self._run(["cat", ver, path])
+
+    def _run_ls(self):
+        ver = self._val(self.ls_ver)
+        if not ver: self._warn("Firmware file is required."); return
```

Ruff (0.15.9)

[error] Multiple statements on one line (colon)

(E701)